### DISCUSSION TITLE
Welcome to flutter_twitter_clone Discussions!

### DISCUSSION THREAD

**Discussion by TheAlphamerc** [TheAlphamerc/flutter_twitter_clone#142] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by OsamaQureshi796** [comment]
Hi.
First of all Thank you for your great work.
Kindly complete it asap because i want to learn it.
